api-client/src/endpoints/createPOIsEndpoint.ts

```diff
@@ -31,6 +31,7 @@ export default (baseUrl: string): Endpoint<ParamsType, Array<PoiModel>> =>
               thumbnail: poi.thumbnail,
               availableLanguages: mapAvailableLanguages(poi.available_languages),
               excerpt: poi.excerpt,
+              metaDescription: poi.meta_description ? poi.meta_description : null,
```

Comment:
You can just write `metaDescription: poi.meta_description ?? null`

Reply:
Does not work as the meta description is an empty string if it is not set and `"" ?? null` resolves to ""